src/helpers/storage.js

```diff
@@ -1,6 +1,8 @@
 // eslint-disable-next-line import/prefer-default-export
 import { STORAGE_KEY } from '../config/consts';
 
+// let Task = { completed: false, id: '', message: '' };
```

Comment:
I wanted to create here Object Task to pass It in function UpdateTaskInStorage (27th line), but apparently it is not neccesary, I do not exactly understand Why it is not neccesary

Reply:
it is not neccesary because you already have this structure in TaskItem component, and you know they are save in that form in localStorage as well. You use function updateTaskInStorage in completely different place, here you are only defining that function.